FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Extensions/JObjectExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Extensions
+{
+    public static class JObjectExtensions
+    {
+        private const string LastUpdatedPropertyName = "lastUpdated";
+
+        /// <summary>
+        /// Extract last update day information from search response bundle.
+        /// </summary>
+        /// <param name="bundle">input bundle.</param>
+        /// <returns>lastupdate timestamp of the bundle.</returns>
+        public static DateTime? GetLastUpdatedDay(this JObject bundle)
```

Comment:
Can we add some unit tests for this extension?

Reply:
Sure. Added